spec/resque/plugins/unique_by_arity_spec.rb

```diff
@@ -29,6 +29,36 @@ def self.perform(_req, _opts = {})
         expect(subject.redis_unique_hash(class: subject.to_s, args: args)).to eq ['ef0f8a28f2c84e48211489121112e67f', [1]]
       end
     end
+
+    context 'with params beyond the unique arity' do
+      let(:instance) do
+        Class.new do
+          def self.to_s
+            'RealFake'
+          end
+
+          def self.perform(_a, _b, _opts = {})
+            # Does something
+          end
+
+          include Resque::Plugins::UniqueByArity.new(
+            arity_for_uniqueness: 1,
+            unique_at_runtime: true,
+            unique_in_queue: true
+          )
```

Comment:
Can you add tests for:
- Not having `arity_for_uniqueness`
- Having `arity_for_uniqueness` with 2